source/common/tracing/http_tracer_impl.h

```diff
@@ -88,6 +88,12 @@ class NullSpan : public Tracing::Span {
   // Tracing::Span
   void setTag(const std::string&, const std::string&) override {}
   void finishSpan() override {}
+  void injectContext(Http::HeaderMap&) override {}
+  SpanPtr spawnChild(const std::string&, SystemTime) override {
```

Comment:
null span had 0 coverage, can you also update your PR to include that?
and make sure that we have full coverage of the new code as well